TotalCrossVM/CMakeLists.txt

```diff
@@ -0,0 +1,620 @@
+cmake_minimum_required(VERSION 3.13.5)
+
+# set the project name
+project(TCVM VERSION 1.0)
+
+if(NOT DEFINED BUILD_SHARED_LIBS AND NOT DEFINED TCVM_SHARED)
+  if(CMAKE_GENERATOR STREQUAL Xcode)
+    set(TCVM_SHARED OFF) 
+    message("tcvm: ${TCVM_SHARED}")  
+  else() # others
+    set(TCVM_SHARED ON) 
+  endif()
+endif()
+
+# Find SDL2 and Skia
+set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/modules")
```

Comment:
I'd move this to the beginning of the file, probably line 4 or 5, as this is generic code and also, this is not restricted to those modules but any modules you ever develop so the comment is misleading.

Reply:
Sounds good to me.